frontend/src/components/core/Sidebar/Sidebar.scss

```diff
@@ -155,3 +155,7 @@
     }
   }
 }
+
+div[data-baseweb="popover"] {
+  z-index: $z-index-sidebar-mobile + 1;
+}
```

Comment:
This is pretty hacky :(  You didn't figure out a better solution for this after all?

Reply:
Here's the solution: https://github.com/streamlit/streamlit/commit/3f4b7fe0c77565b5c0ff716ab65d85f1d3be9aa2#diff-48d8dc927a0e0b14d84f8f380fe19b80L32

Reply:
Ok I just pushed a fix into this PR :smiley: